.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 sudo: required  # See http://docs.travis-ci.com/user/trusty-ci-environment/
 dist: trusty
 node_js:
-  - "4"
+  - "6"
```

Comment:
I know what an LTS is, but don't understand what this means for our repository, or what Erwin's thumbs up means :)

Do we want to stick with node 4.0? This means that `yarn.lock` will be permanently out of date, given that we cannot update it until we move off of node 4.0. Is that acceptable? Won't new committers run into dependency problems while developing because yarn.lock is out of date?